README.md

```diff
@@ -481,7 +481,9 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Bandwidth](https://boards.greenhouse.io/bandwidth/jobs/4618588) | Rochester, NY | Software Engineering Intern (Summer 2023) |
 | [UKG](https://careers.ukg.com/careers/JobDetail/Software-Engineering-Internship-Spring-Summer-2023/41891) | Multiple | Software Engineering Internship – Spring & Summer 2023 (No Sponsorship) | 
 | [MediaTek](https://careers.mediatek.com/eREC/JobSearch/JobDetail/MUS120210917004?langKey=en-US) | San Jose, CA | AI Architecture and Algorithm Intern |
-
+| [Citizens](https://jobs.citizensbank.com/job/-/-/288/36919252544) | Johnston, Rhode Island; Phoenix, Arizona | Technology Summer Internship Program: Software Engineer (No Sponsorship) |
+| [Philips](https://philips.wd3.myworkdayjobs.com/jobs-and-careers/job/Bothell/Software-Engineering-Intern_478729) | Bothell, WA | Software Engineering Intern (No Sponsorhip) |
+| [Schweitzer Engineering Laboratories](hhttps://selinc.wd1.myworkdayjobs.com/en-US/SEL/job/Washington---Pullman/Software-Engineer-Intern_2022-10210-1) | Washington - Pullman | Software Engineer Intern |
```

Comment:
```suggestion
| [Schweitzer Engineering Laboratories](https://selinc.wd1.myworkdayjobs.com/en-US/SEL/job/Washington---Pullman/Software-Engineer-Intern_2022-10210-1) | Washington - Pullman | Software Engineer Intern |
```

Reply:
Doesn't this mean no sponsorship?

<img width="742" alt="image" src="https://user-images.githubusercontent.com/37476886/197671396-8e23b50a-e55b-4d75-aa9d-20490f60e1f8.png">

Reply:
That does not necessarily imply no sponsorship. That being said, feel free to make a (dummy) application and see if they automatically reject you based on the need for a sponsorship.

Reply:
they did 🥹